tensorflow/core/util/ctc/ctc_beam_search.h

```diff
@@ -304,7 +307,7 @@ void CTCBeamSearchDecoder<CTCBeamState, CTCBeamComparer>::Step(
       b->newp.label += raw_input(b->label) - norm_offset;
     }
     // Pblank(l=abc @ t=6) = P(l=abc @ t=5) * P(- @ 6)
-    b->newp.blank = b->oldp.total + raw_input(blank_index_) - norm_offset;
+    b->newp.blank = b->oldp.total + raw_input(this->blank_index_) - norm_offset;
```

Comment:
why?

Reply:
same as in https://github.com/tensorflow/tensorflow/pull/31164#discussion_r312368512